content/id/docs/concepts/overview/object-management-kubectl/declarative-config.md

```diff
@@ -0,0 +1,868 @@
+tentang definisi dan konfigurasi objek-objek Kubernetes. Jika belum pernah, kamu disarankan untuk membaca terlebih dulu dokumen-dokumen berikut:
+
+- [Pengelolaan Objek Kubernetes Menggunakan Perintah Imperatif](/docs/concepts/overview/object-management-kubectl/imperative-command/)
+- [Pengelolaan Objek Kubernetes Menggunakan File Konfigurasi Imperatif](/docs/concepts/overview/object-management-kubectl/imperative-config/)
+
+Berikut adalah beberapa defnisi dari istilah-istilah yang digunakan
+dalam dokumen ini:
+
+- *objek file konfigurasi / file konfigurasi*: Sebuah *file* yang
+mendefinisikan konfigurasi untuk sebuah objek Kubernetes. Dokumen ini akan memperlihatkan cara menggunakan *file* konfigurasi dengan perintah `kubectl apply`. *File-file* konfigurasi biasanya disimpan di sebuah *source control* seperti Git.
+- *konfigurasi objek live / konfigurasi live*: nilai konfigurasi *live* dari sebuah objek, sebagaimana yang tersimpan di kluster Kubernetes. Nilai-nilai ini disimpan di *storage* kluster Kubernetes, biasanya etcd.
+- *writer konfigurasi deklaratif / writer deklaratif*: Seseorang atau sebuah komponen perangkat lunak yang membuat pembaruan ke objek *live*. *Live writer* yang disebut pada dokumen ini adalah *writer* yang membuat perubahan terhadap *file* konfigurasi objek dan menjalankan perintah `kubectl apply` untuk menulis perubahan-perubahan tersebut.
+
+## Cara membuat objek
+
+Gunakan perintah `kubectl apply` untuk membuat semua objek, kecuali objek-objek yang sudah ada sebelumnya, yang didefinisikan di *file-file* konfigurasi dalam direktori yang ditentukan:
+
+```shell
+kubectl apply -f <directory>/
+```
+
+Perintah di atas akan memberi anotasi `kubectl.kubernetes.io/last-applied-configuration: '{...}'` pada setiap objek yang dibuat. Anotasi ini berisi konten dari *file* konfigurasi objek yang digunaan untuk membuat objek tersebut.
+
+{{< note >}}
+Tambahkan parameter `-R` untuk memproses seluruh direktori secara rekursif.
+{{< /note >}}
+
+Berikut sebuah contoh *file* konfigurasi objek:
+
+{{< codenew file="application/simple_deployment.yaml" >}}
+
+Jalankan perintah `kubectl diff` untuk menampilkan objek yang akan dibuat:
+
+```shell
+kubectl diff -f https://k8s.io/examples/application/simple_deployment.yaml
+```
+
+{{< note >}}
+Perintah `diff` menggunakan [*server-side dry-run*](/docs/reference/using-api/api-concepts/#dry-run), yang perlu diaktifkan di `kube-apiserver`.
+{{< /note >}}
+
+Buat objek dengan perintah `kubectl apply`:
+
+```shell
+kubectl apply -f https://k8s.io/examples/application/simple_deployment.yaml
+```
+
+Tampilkan konfigurasi *live* dengan perintah `kubectl get`:
+
+```shell
+kubectl get -f https://k8s.io/examples/application/simple_deployment.yaml -o yaml
+```
+
+Keluaran perintah di atas akan menunjukkan bahwa anotasi `kubectl.kubernetes.io/last-applied-configuration` sudah dituliskan ke konfigurasi *live*, dan anotasi tersebut sesuai dengan *file* konfigurasi:
```

Comment:
Sudah dituliskan -> sudah diterapkan?

Reply:
The original English words are:
The output shows that the kubectl.kubernetes.io/last-applied-configuration annotation was written to the live configuration, and it matches the configuration file.

I think "dituliskan" is ok for this.